index.js

```diff
@@ -27,6 +27,13 @@ exports.register = function(server, options, next) {
   const renderHandler = function(viewConfig) {
     return function(request, reply) {
       async.autoInject({
+        preProcess: done => {
+          if (typeof options.preProcess === 'string') {
```

Comment:
dont need to check if its a string, str2fn will call it if it's a function - like if somebody was using outside hapi-confi